### ISSUE TITLE
Crash with error (0.3.2)

### ISSUE THREAD

**Issue by caffeine-mgn** [allusive-dev/picom-allusive#9]
Kernel: 6.2
OS: Majaro
CPU: amd 5950X
GPU: Nvidia 3090
DE: Awesome 4.3

```
[ 15.10.2023 14:08:02.077 glx_bind_pixmap ERROR ] Failed to query info of pixmap 0x01e00fea.
[ 15.10.2023 14:08:02.077 win_paint_shadow ERROR ] Window 0x002220a2 is missing shadow data.
```

Happens randomly. More often immediately after changing config file

[picom.conf.txt](https://github.com/allusive-dev/picom-allusive/files/12909379/picom.conf.txt)
My config

**Comment by allusive-dev** [comment]
Firstly can I ask why you created 2 issues and would you be able to merge them please. 
Second I can't help right now as it is late but I will try to see what I can do about it in around 10 hours.

**Comment by caffeine-mgn** [comment]
Looks like two different problems.

The problem is not critical.

**Comment by allusive-dev** [comment]
Can you try these please and let me know how it goes.

Remove everything to do with transistions in your config as I am pretty sure this fork dosent implement that.

Second try just running the sample configuration and let me know if there are any issues or the same issues for either of these.

**Comment by allusive-dev** [comment]
@caffeine-mgn This issue should hopefully be resolved as of `v1.2` or later.

Let me know if that is not the case.
